### PR TITLE
refactor(recipe): Updated types in recipe and added additional fields

### LINKED ISSUE #7: Update Recipe Model
Found a [recipe](https://www.maischemalzundmehr.de/index.php?id=1793&inhaltmitte=rezept) which contains more fields and more hops (7 instead of currently 3 we do have in our model). Therefore, we need to update the model, to be able to represent everything.

### PR DESCRIPTION
**Pull request by tdankert**
Updated recipe model, based on findings of a recipe (additional keys in json and different types for some values (e.g. int instead of string))

closes #7

### CODE REVIEW

**Comment by tdankert** [comment]
@Johannes123123412341412412412412414 we do need a translation for "Stopfhopfen". Could you provide this? I've got no idea, what that even means ;)

**Comment by Johannes123123412341412412412412414** [comment]
I think you are referring to "hopfenstopfen", that would be dry hopping.

**Comment by tdankert** [comment]
> I think you are referring to "hopfenstopfen", that would be dry hopping.

Maybe. It's about [this](recipe)

File: Brauser.Common/Recipe/Hop.cs [hunk 1]
```diff
@@ -5,6 +5,6 @@ public record Hop
     public int? Order { get; set; }
     public string? Variety { get; set; }
     public double? Quantity { get; set; }
-    public int? Alpha { get; set; }
-    public DateTimeOffset? CookingTime { get; set; }
+    public double? Alpha { get; set; }
+    public int? CookingTime { get; set; }
```
Comment (n-develop) [inline Brauser.Common/Recipe/Hop.cs:9]: Do we really want an `int` here? It's our business model, so we can decide if we want to convert it to something useful like `DateTimeOffset` after importing the MaischeMalzUndMehr recipe.

File: nuget.config [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (n-develop) [inline nuget.config:1]: Love it.

**Review by n-develop — APPROVED** [review]
Just one suggestion regarding the data type

File: Brauser.Common/Recipe/Hop.cs (hunk 1 above)
Comment (tdankert) [reply Brauser.Common/Recipe/Hop.cs:9]: Fair point. Maybe we could rename it in CookingTimeInMinutes.
DateTimeOffset is nice, but it seems we exceed an hour in very few occasions and seems a little bit overkill in that case.

Comment (n-develop) [reply Brauser.Common/Recipe/Hop.cs:9]: `DateTimeOffset` would not be a good candidate, as it is just a date and says nothing about the duration. My proposal would be `TimeSpan` here. But `CookingTimeInMinutes` would be fine as well. 👍

File: nuget.config (hunk 2 above)
Comment (tdankert) [reply nuget.config:1]: This is my favorite. No idea, what's going on. Maybe encoding stuff.

File: Brauser.Common/Recipe/Hop.cs (hunk 1 above)
Comment (tdankert) [reply Brauser.Common/Recipe/Hop.cs]: Didn't have TimeSpan on the screen. Nice!

**Comment by Johannes123123412341412412412412414** [comment]
> > I think you are referring to "hopfenstopfen", that would be dry hopping.
> 
> Maybe. It's about [this recipe](https://www.maischemalzundmehr.de/index.php?id=1793&inhaltmitte=rezept)

Alright, the noun is "Dry Hop".

**Comment by tdankert** [comment]
> > > I think you are referring to "hopfenstopfen", that would be dry hopping.
> > 
> > 
> > Maybe. It's about [this recipe](https://www.maischemalzundmehr.de/index.php?id=1793&inhaltmitte=rezept)
> 
> Alright, the noun is "Dry Hop".

Thx. just updated it

**Pull request merged**